test/unit/tap-reporter.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-env mocha */
+
+var chai = require('chai')
+var sinon = require('sinon')
+var EventEmitter = require('events').EventEmitter
+var JsReporters = require('../../dist/js-reporters.js')
+var data = require('./data.js')
+var expect = chai.expect
+
+chai.use(require('sinon-chai'))
+
+describe('Tap reporter', function () {
+  var emitter
+
+  before(function () {
+    emitter = new EventEmitter()
+    var reporter = new JsReporters.TapReporter(emitter)
```

Comment:
That indicates that we shouldn't be using a constructor here. Calling `JsReporters.TapReporter` without `new` seems fine.